.changeset/popular-squids-kneel.md

```diff
@@ -0,0 +1,5 @@
+---
+'@clerk/clerk-js': patch
+---
+
+TODO
```

Comment:
We probably only need an empty changeset for an internal change like this one :)

Reply:
I think we should bump the version because this upgrade contains breaking changes for `floating-ui` and thus can subtly change how things behave for our users.

While I did my extensive check into the changelog it still might change behavior and with having a release we can bisect it better. Publishing doesn't hurt here :)

Reply:
Thanks for explaining Lennart. Everything makes sense to me, let's go ahead and merge this one